Assets/Character/Scripts/AnimatorCommon.cs

```diff
@@ -32,8 +32,19 @@ public static void SetState(Animator animator, int val)
         animator.SetInteger("state", val);
     }
 
+    public static void RestartCurrentAnimation(Animator animator)

```

Comment:
Consider just making a wrapper for Animator that includes all these functions since they all take an Animator object